hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/ReceiverTasklet.java

```diff
@@ -102,10 +100,12 @@
 
     //                 END FLOW-CONTROL STATE
 
-    public ReceiverTasklet(OutboundCollector collector, int rwinMultiplier, int flowControlPeriodMs) {
+    public ReceiverTasklet(OutboundCollector collector, int rwinMultiplier, int flowControlPeriodMs,
+                           LoggingService loggingService) {
         this.collector = collector;
         this.rwinMultiplier = rwinMultiplier;
         this.flowControlPeriodNs = (double) MILLISECONDS.toNanos(flowControlPeriodMs);
+        this.logger = loggingService.getLogger(getClass());
```

Comment:
how does this add context? for example, which edge this receiver is corresponding to?

Reply:
It tells which instance it belongs to.

Reply:
Is it enough? Each edge has a different ReceiverTasklet instance.